src/openvic-simulation/country/CountryInstance.hpp

```diff
@@ -98,6 +102,10 @@ namespace OpenVic {
 
 		/* Budget */
 		fixed_point_t PROPERTY(cash_stockpile);
+		slider_value_t PROPERTY(poor_tax_rate_slider);
+		slider_value_t PROPERTY(middle_tax_rate_slider);
```

Comment:
Poor/middle/rich strata aren't hardcoded in OpenVic, they're dynamically loaded like most other game data. This means we can't just have a regular variable for each starta's tax rate, instead we need a map from `Strata` to `slider_value_t` (or whatever name it ends up with), specifically `IndexedMap<Strata, slider_value_t>`, only requiring the vector of loaded strata as input which can be passed to it similar to how all the other `IndexedMap` key vectors are passed around